recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python -m pip install --no-deps .
```

Comment:
Sorry, I should be more clear. It wasn't obvious to me that we had chosen to go for `pip` installs now. @ocefpaf, care to comment.